src/IdentityServer/Configuration/DependencyInjection/Options/LoggingOptions.cs

```diff
@@ -22,7 +22,6 @@ public class LoggingOptions
     public ICollection<string> BackchannelAuthenticationRequestSensitiveValuesFilter { get; set; } =
         new HashSet<string>
         {
-            // TODO: IdentityModel
             OidcConstants.TokenRequest.ClientSecret,
             OidcConstants.TokenRequest.ClientAssertion,
             OidcConstants.AuthorizeRequest.IdTokenHint
```

Comment:
I think this is enough of a filter, but I imagine some organizations might also want to remove login_hint and login_hint_token, because maybe email address is used as the username, and email addresses are PII, thus not allowed in logs. I think letting people add that to the filter config if they need it is fine. Thoughts?